main.tf

```diff
@@ -130,14 +130,14 @@ resource "aws_cloudfront_distribution" "this" {
     }
   }
 
-  #dynamic "logging_config" {
-  #  for_each = var.logging_config[*]
-  #  content {
-  #    bucket          = lookup(logging_config.value,"bucket",null)
-  #    include_cookies = lookup(logging_config.value,"include_cookies",null)
-  #    prefix          = lookup(logging_config.value,"prefix",null)
-  #  }
-  #}
+  dynamic "logging_config" {
+   for_each = var.logging_config[*]
```

Comment:
I guess there's one space of indentation missing for the block below `dynamic`

Reply:
Fixed